### ISSUE TITLE
enhancement: appium driver/plugin list should show package name for known extensions

### ISSUE THREAD

**Issue by boneskull** [appium/appium#16798]
Running `appium driver list` or `appium plugin list` displays the "short" driver name, which is good, but it'd also be helpful if the package name was also displayed (linked via URL, if supported by terminal).

```
✔ Listing available drivers
- fake@3.2.6 [installed (linked from /Users/chrishiller/projects/appium/packages/fake-driver)]
- uiautomator2 [not installed]
- xcuitest [not installed]
- youiengine [not installed]
- windows [not installed]
- mac [not installed]
- mac2 [not installed]
- espresso [not installed]
- tizen [not installed]
- flutter [not installed]
- safari [not installed]
- gecko [not installed]
```

I wanted to use this list to grab a working copy of e.g., the gecko driver, but I didn't know the package name, so I didn't have easy access to the URL.

If we consider this to be too much information for end users, perhaps a `--details` or `--verbose` flag would be appropriate here; we could list the npm package name and the repository URL?

**Comment by jlipps** [comment]
yeah, agree this should be available somewhere, a --details or --verbose would be a good place to hide it

**Comment by mykola-mokhnach** [comment]
closed as fixed
